plugins/woocommerce/tests/legacy/unit-tests/rest-api/Tests/Version2/product-reviews.php

```diff
@@ -193,6 +197,8 @@ public function test_create_product_review() {
 				'date_created'         => $data['date_created'],
 				'date_created_gmt'     => $data['date_created_gmt'],
 				'product_id'           => $product->get_id(),
+				'product_name'           => $product->get_name(),
+				'product_permalink'           => $product->get_permalink(),
```

Comment:
Looks like some whitespace issues here

Reply:
Fixed in [7603a2c](https://github.com/woocommerce/woocommerce/pull/32073/commits/7603a2c02b42a54272847067225c18d5b1ff6093)